### ISSUE TITLE
Add MultipartFormData to Swift 2.0 branch

### ISSUE THREAD

**Issue by mcudich** [Alamofire/Alamofire#583]
The feature was merged into master, but is not yet ported over to Swift 2.0. Thanks in advance!

**Comment by cnoon** [comment]
You are absolutely right @mcudich. We are trying to finish up the 1.3.0 release before we starting porting all the changes over. There are some major features (MultipartFormData included) that will require a significant amount of work. We will start moving the changes over to the `swift-2.0` branch as quickly as we can.
